Tracking3D/Tracking3D.py

```diff
@@ -0,0 +1,505 @@
+from typing import Optional
+
+import slicer
+import vtk
+from slicer import vtkMRMLScalarVolumeNode, vtkMRMLSequenceNode, vtkMRMLTransformNode
+from slicer.i18n import tr as _
+from slicer.parameterNodeWrapper import parameterNodeWrapper
+from slicer.ScriptedLoadableModule import (
+    ScriptedLoadableModule,
+    ScriptedLoadableModuleLogic,
+    ScriptedLoadableModuleWidget,
+)
+from slicer.util import VTKObservationMixin
+from Tracking3DLib.TreeNode import TreeNode
+
+import AutoscoperM
+
+try:
+    # Not a big fan of checking if Elastix is installed this way...
+    # But running hasattr(itk, "ElastixRegistrationMethod") is PAINFULLY slow
```

Comment:
```suggestion
    # Since running hasattr(itk, "ElastixRegistrationMethod") is slow,
    # check if Elastix is installed by attempting to import ElastixRegistrationMethod
```

Reply:
To avoid impacting loading of modules, we should move the install in the logic.

For an example:
* https://github.com/SlicerMorph/SlicerANTs/blob/main/ITKANTsCommon/ITKANTsCommon.py
* https://github.com/SlicerMorph/SlicerMorph/blob/5a833a8efc5d8ae123a34f7da1eb2f5125c6b496/ALPACA/ALPACA.py#L92